atest/testdata/keywords/type_conversion/keyword_decorator_with_wraps.robot

```diff
@@ -0,0 +1,10 @@
+*** Settings ***
+Force Tags        require-py3
+Library           LibraryWithKeywordWraps.py
+
+*** Test Cases ***
+Keyword Decorator With Wraps
+    Keyword With Wraps    42
+
+Keyword Decorator With Wraps Mismatched Type
+    Run Keyword And Expect Error    *Argument 'arg' got value 'string' that cannot be converted to integer.    Keyword With Wraps    string
```

Comment:
We've used `[Documentation]` to specify the expected error instead of using `Run Keyword And Expect Error`.